packages/attestation-service/src/sms/messagebird.ts

```diff
@@ -61,12 +64,36 @@ export class MessageBirdSmsProvider extends SmsProvider {
     return [bodyParser.urlencoded({ extended: false })]
   }
 
+  async getUSNumbers(): Promise<string[]> {
```

Comment:
didn't they mentioned this is the case for canadian numbers as well? or only US numbers which are required for sending to US/Canada?